Packs/jamf/ReleaseNotes/2_2_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### Jamf Modeling Rule
+
+Updated the Parsing Rule logic to consider the webhook field with NULL values.
```

Comment:
Seems the issue was the usage of the regex operator on the JSON object rather than having null values, 
So I would adjust the RN accordingly, perhaps something like: 
`Fixed an issue with the timestamp format validation. `

Reply:
The regex received NULL, the issue wasn't with the regex...